cmd/init/config.go

```diff
@@ -11,48 +11,56 @@ import (
 )
 
 func generatehcl2NixConf(pt langdetect.ProjectType, pd *langdetect.ProjectDetails) (hcl2nix.Config, error) {
+	fmt.Println("IN GENERATE HCL NIX CONFIG -------------------------------------------")
+	fmt.Println("NAME: ", pd.Name)
 	switch pt {
 	case langdetect.GoModule:
 		return genGoModuleConf(pd), nil
 	case langdetect.PythonPoetry:
-		return genPythonPoetryConf(), nil
+		return genPythonPoetryConf(pd), nil
 	case langdetect.RustCargo:
-		config, err := genRustCargoConf()
+		config, err := genRustCargoConf(pd)
 		if err != nil {
 			return hcl2nix.Config{}, err
 		}
 		return config, nil
 	case langdetect.JsNpm:
-		config, err := genJsNpmConf()
+		config, err := genJsNpmConf(pd)
 		if err != nil {
 			return hcl2nix.Config{}, err
 		}
 		return config, nil
 	default:
-		return generateEmptyConf(), nil
+		return generateEmptyConf(pd), nil
 	}
 }
 
-func generateEmptyConf() hcl2nix.Config {
+func generateEmptyConf(pd *langdetect.ProjectDetails) hcl2nix.Config {
```

Comment:
This shouldn't accept `pd`. It should only accept `name string`  where name is the image name